src/imagetilesource.js

```diff
@@ -50,6 +50,7 @@
      * @extends OpenSeadragon.TileSource
      * @param {Object} options Options object.
      * @param {String} options.url URL of the image
+     * @param {String} options.referenceStripThumbnailUrl URL of the dedicated thumbnail image
```

Comment:
This new property should work for all tile sources, so the documentation shouldn't just be in ImageTileSource. I'm kind of on the fence as to whether it should appear in every single tile source file, or just in the root tilesource.js. 

Also, you should add something like "to be used by the reference strip" just to be clear.

Otherwise this patch is looking ready to go!